File: java/home_work_1/task9.java
import java.util.*;
class Task1 {
    public static void main(String[] args) {
        
            int x;
	    int sum = 0;
	    System.out.println("Please Enter number");
            Scanner sx = new Scanner(System.in);
	    x = sx.nextInt();
	    do {
		int r = x % 10;
		sum += r;
		x = (x - r) / 10;
	    } while (x != 0);

		System.out.println( "Result is " + sum );
    }
}

File: cpp/home_work_1/task9.cpp
#include <iostream>

int main() {
    int x;
    int sum = 0;

    std::cout << "Please enter x ";
    std::cin >> x;

    do {
        int r = x % 10;
        sum += r;
        x = (x - r) / 10;
    } while (x != 0);

    std::cout << "Result is " << sum << std::endl;

    return 0;
}
